src/tools/perf/lib/libperf_int.h

```diff
@@ -231,6 +231,47 @@ size_t ucx_perf_get_message_size(const ucx_perf_params_t *params)
     return length;
 }
 
+typedef struct {
+    ucs_status_t (*setup)(ucx_perf_context_t *perf);
+    void         (*cleanup)(ucx_perf_context_t *perf);
+    ucs_status_t (*run)(ucx_perf_context_t *perf);
+    void         (*barrier)(ucx_perf_context_t *perf);
+} ucx_perf_funcs_t;
+
+extern ucx_perf_funcs_t ucx_perf_funcs[];
```

Comment:
order in H file should be:
1. defines
2. enums
3. structs
4. extern (global vars)
5. func prototypes
6. inline funcs